lib/postprocessing/math.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+class Dimensions {
+    constructor(width, height) {
+        // if width is Dimension or Dimension like, clone it
+        if (width && (width.width || width.height)) {
```

Comment:
Maybe rather check if width is an object? It would make it slightly more understandable - because testing `width.heigth` needs some brain-parsing. Wdyt?

Reply:
if `width.width` (or `width.height`) is true, it is an object. And we don't care about what it is, as long as it has width or height properties. If it quacks like a duck...